projects/plutus/inc/ltc_afe.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+// driver for LTC6804-1 AFE chip
```

Comment:
Should probably explicitly point out that this assumes we have a 16 channel analog mux attached to the GPIO with 4 as channel select outputs and 1 as a thermistor input.